apps/src/templates/studioHomepages/ActionBlocksWrapper.story.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ActionBlocksWrapper from './ActionBlocksWrapper';
+
+export default {
+  component: ActionBlocksWrapper,
+};
+
+const generateActionBlocks = tileCount => {
+  const actionBlocks = [];
+  for (let i = 0; i < tileCount; i++) {
+    actionBlocks.push({
+      overline: `Overline ${i + 1}`,
```

Comment:
Thank you for using the prop name in the text here! I wasn't sure what an overline was and it was so useful to see the screenshot and immediately understand!